### PR TITLE
Bugfix/77 bug context issue

### PR DESCRIPTION
**Pull request by JWebCoder**
updated some context logic along with the trigger of the move actions
ready for testing

### CODE REVIEW

File: packages/lumina-core/src/components/tabs/treeView/index.tsx [hunk 1]
```diff
@@ -39,31 +39,21 @@ export const TreeViewTab = () => {
...
         //perfect match
-        if (friendlyNameLower === searchValLower) matches = true
-        // name starts with
-        else if (friendlyNameLower.startsWith(searchValLower)) matches = true
-        // name contains part of the search
-        else if (friendlyNameLower.includes(searchValLower)) matches = true
-
-        // boolean check to signal the filtered elements
-        if (matches) el.isMatch = true
+        console.log(friendlyNameLower, searchValLower, friendlyNameLower.includes(searchValLower))
```
Comment (bmirra01) [inline packages/lumina-core/src/components/tabs/treeView/index.tsx:45]: This should be removed before merge I believe

**Review by bmirra01 — DISMISSED** [review]

**Comment by bmirra01** [comment]
Bug: when adding a component (using add component button) it does add on AppContext, but on Treeview it doesn't update, even if you click on 'filter' button (like we used to do to bypass the state update)

**Review by bmirra01 — CHANGES_REQUESTED** [review]
There is that line to remove, and the bug question I've raised

File: packages/lumina-core/src/components/tabs/treeView/index.tsx (hunk 1 above)
Comment (JWebCoder) [reply packages/lumina-core/src/components/tabs/treeView/index.tsx]: fixed

**Comment by JWebCoder** [comment]
> Bug: when adding a component (using add component button) it does add on AppContext, but on Treeview it doesn't update, even if you click on 'filter' button (like we used to do to bypass the state update)
> 
> @JWebCoder

fixed

**Comment by bmirra01** [comment]
That is fixed, a new bug shows up now:
image

What's happening now, is that, you add the component on AppContext, the new component shows at the end of the array in AppContext, but on Treeview, you are moving things independently, is like the state is not updating the AppContext

**Review by bmirra01 — CHANGES_REQUESTED** [review]
Check my message above about the new bug
@JWebCoder

**Comment by JWebCoder** [comment]
Fixed: https://github.com/Funfunz/lumina-next/pull/82#issuecomment-2321736769

**Review by bmirra01 — APPROVED** [review]
Looks good now :-)

**Pull request merged**